src/components/icon/index.stories.mdx

```diff
@@ -0,0 +1,55 @@
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs";
```

Comment:
`Preview` and `Props` were renamed to `Canvas` and `ArgsTable` respectively.  Can you update this?

Reply:
Updated